taf/src/main/java/com/epam/tat/framework/util/StringUtils.java

```diff
@@ -0,0 +1,10 @@
+package com.epam.tat.framework.util;
+
+import java.util.Arrays;
+
+public class StringUtils {
+
+    public static String removeCommas(String[] args) {
```

Comment:
It looks to me that wrapping this one-liner method is not absolutely necessary. Also as an alternative, you may consider Apache Utils Lang StringUtils. https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html